src/Http/Routing/src/Matching/AcceptsMatcherPolicy.cs

```diff
@@ -0,0 +1,392 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Headers;
+using Microsoft.AspNetCore.Http.Metadata;
+
+namespace Microsoft.AspNetCore.Routing.Matching;
+
+internal sealed class AcceptsMatcherPolicy : MatcherPolicy, IEndpointComparerPolicy, INodeBuilderPolicy, IEndpointSelectorPolicy
```

Comment:
Is this meant to replace ConsumesMatcherPolicy from MVC?

Reply:
yes. we renamed ConsumesMatcherPolicy to AcceptsMatcherPolicy